### PR TITLE
define a nav section in the main index

### PR DESCRIPTION
**Pull request by charislam**
# Description

Make a nav section in the main index containing pages that help with finding info.

# Links

Related to https://github.com/timescale/docs-private/issues/64

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

**Review by Loquacity — APPROVED** [review]
Yes, I like this! Let's merge and get some feedback 👍

File: navigation/page-index/page-index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+module.exports = [
+  {
+    title: "Find a docs page",
+    href: "navigation",
+    filePath: "index.md",
+    children: [
+      {
+        title: "Find pages by Cloud Console location",
```
Comment (Loquacity) [inline navigation/page-index/page-index.js:8]:
```suggestion
        title: "Find by Cloud Console location",
```

Just for parallelism.

File: navigation/page-index/page-index.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      },
+      {
+        title: "Find by keyword",
+        overrideHref: "/keywords",
+        excerpt: "Browse topics by keywords.",
+        type: "placeholder",
+      },
+      {
+        title: "Full search",
+        overrideHref: "/search",
```
Comment (Loquacity) [inline navigation/page-index/page-index.js:21]: When you click this in preview, it gives you a no results page. Can we preload it with a search term? "Timescale" maybe?

Comment (charislam) [reply navigation/page-index/page-index.js:21]: yeah I'm going to redo the empty state 👍🏼

Comment (Loquacity) [reply navigation/page-index/page-index.js:21]: Excellent 😁

**Pull request merged**